p2panda-js/src/wasm.test.ts

```diff
@@ -121,37 +118,37 @@ describe('Web assembly interface', () => {
       // Create message
       const fields = new MessageFields();
       fields.add('description', 'str', 'Hello, Panda');
-      expect(fields.get('description')).to.eq('Hello, Panda');
+      expect(fields.get('description')).toBe('Hello, Panda');
 
       const messageEncoded = encodeCreateMessage(TEST_SCHEMA, fields);
 
       // Sign and encode entry
       const { entryEncoded, entryHash } = signEncodeEntry(
         keyPair,
         messageEncoded,
-        null,
-        null,
+        undefined,
+        undefined,
```

Comment:
I would be interested to hear why you changed this.

Reply:
(the tests still pass for me with `null`, which is more idiomatic in my understanding of JavaScript)

Reply:
I see the following error here, not sure why this is different from what you can see?

> src/wasm.test.ts:129:9 - error TS2345: Argument of type 'null' is not assignable to parameter of type 'string | undefined'.

Reply:
The error occurs when I run the tests .. it must be the `ts-jest` plugin giving me that warning? When I do `npm run build` I don't get warned, when I run `./node_modules/.bin/tsc` I do get it ..

Reply:
Also, indeed the `index.d.ts` file in `wasm-node` declares the types as following: 

```typescript
export function signEncodeEntry(key_pair: KeyPair, encoded_message: string, entry_skiplink_hash: string | undefined, entry_backlink_hash: string | undefined, seq_num: number, log_id: number): any
```

Reply:
Strange issue, I didn't get any of these issues.